exonum-dictionary.txt

```diff
@@ -21,18 +21,17 @@ clonned
 codecov
 compat
 concat
+CORS
```

Comment:
Since you have already touched it.
AFAIK cspell is case insensitive, so it can be removed